### PR TITLE
Add Flag: --ignore-warnings

### PR DESCRIPTION
**Pull request by pascalholthaus**
**What**: Add Flag: --ignore-warnings

**Why**:

**How**:

**Checklist**:

- [ ] Tests
- [ ] Conventional commit message
- [ ] Documentation

### CODE REVIEW

File: tests/plugins/test_files/nasl/waring.nasl [hunk 1]
```diff
@@ -0,0 +1,68 @@
+CPE = "cpe:/a:apache:http_server";
```
[the remark this answers is not in the document]
Comment (pascalholthaus) [reply tests/plugins/test_files/nasl/waring.nasl]: Renamed, but i need this error.

File: troubadix/reporter.py [hunk 2]
```diff
@@ -95,18 +97,22 @@ def _process_plugin_results(
...
-                    report = self._report_ok
+                    if self._verbose > 0:
+                        self._report_ok(plugin_result.message)
                 elif isinstance(plugin_result, LinterResult):
-                    report = self._report_ok
-
-                if self._verbose > 0:
-                    report(plugin_result.message)
+                    if self._verbose > 0:
+                        self._report_ok(plugin_result.message)
```
[the remark this answers is not in the document]
Comment (pascalholthaus) [reply troubadix/reporter.py:115]: Better an second if over line 109?

File: troubadix/argparser.py [hunk 3]
```diff
@@ -77,6 +77,10 @@ def parse_args(
         ),
     )
 
+    parser.add_argument(
+        "--ignore-warnings", action="store_true", help="Set warnings be quiet."
```
[the remark this answers is not in the document]
Comment (pascalholthaus) [reply troubadix/argparser.py]: Done

File: troubadix/reporter.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (y0urself) [reply troubadix/reporter.py:115]: na I don't care. I just wondered why you have switched to multiple `if self._verbose` calls. I guess it is more performant if you put the `if self._verbose > 0:` out and the other `isinstance` if inside.
```python
if self._verbose > 0:
  if isinstance(...):
    ...
```

**Review by y0urself — APPROVED** [review]

**Pull request merged**